packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -1,28 +1,232 @@
 import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
-import type { InputSlots, InputState } from './Input.types';
+import type { InputState } from './Input.types';
+import type { Theme } from '@fluentui/react-theme';
 
-/**
- * Styles for the root slot
- */
-const useStyles = makeStyles<keyof InputSlots | 'root'>({
-  root: theme => ({}),
-  input: theme => ({}),
-  inputWrapper: theme => ({}),
-  bookendBefore: theme => ({}),
-  bookendAfter: theme => ({}),
-  insideStart: theme => ({}),
-  insideEnd: theme => ({}),
+// TODO(sharing) use theme values once available
+const horizontalSpacing = {
+  xxs: '2px',
+  xs: '4px',
+  sNudge: '6px',
+  s: '8px',
+  mNudge: '10px',
+  m: '12px',
+};
+const contentSizes = {
+  // TODO(sharing) shouldn't these be in the theme?
```

Comment:
Is it the same as `react-text/src/typographyStyles/typographyStyles.ts`? Should we reuse `typographyStyles`?

Reply:
It's similar but doesn't quite cover all the cases--the design for Input specifies `large` as using `400` size but I'm pretty sure it still uses regular font weight. Though it's possible that was an oversight. (Incidentally...WHY did we have to use numeric size names that look like font weights...?? I know this is more of a complaint for design and probably too late. 😞)

Reply:
Also the individual typography styles actually aren't exported at the moment. Should we change that (@andrefcdias)? In this case it's not feasible/desirable to use the full text components like `Body`. (Either way, since it's a change in a separate package I think I'll leave this as-is for this PR.)